samples/TR/W3C.css

```diff
@@ -12,13 +12,13 @@ body {
 	padding: 0 !important;
 }
 @page:left {
-	margin: 10% 10% 10% 14%!important;
+	margin: 10% 10% 10% 10% !important;
```

Comment:
Making the page symetrical sounds reasonable, but then I suggest also centering the page number, by putting it in `@bottom-center` instead of `@bottom-left-corner` or `@bottom-right-corner` depending on the page.